mon-pix/app/components/resume-campaign-banner.js

```diff
@@ -3,21 +3,20 @@ import Component from '@glimmer/component';
 
 export default class ResumeCampaignBanner extends Component {
 
-  get unsharedCampaignParticipations() {
-    return this.args.campaignParticipations.filter((campaignParticipation) => campaignParticipation.isShared === false);
+  get unsharedProfileCollectionCampaignParticipations() {
+    return this.args.campaignParticipations.filter((campaignParticipation) => campaignParticipation.isShared === false
+      && campaignParticipation.campaign.get('isProfilesCollection'));
   }
 
   get lastUnsharedCampaignParticipation() {
```

Comment:
```suggestion
  get lastUnsharedProfileCollectionCampaignParticipation() {
```
**Suggestion :** Afin de mieux retranscrire le résultat de la liste que l'on filtre

Reply:
**suggestion (non-blocking) :** Hum, je suis pas complètement d'accord le nommage doit exposer l'intention pas l'implémentation.
Ici l'intention c'est de récupérer la participation à reprendre / finir, et l'implémentation c'est de prendre c'est la plus récente. On pourrait changer l'implémentation et avoir la même intention : On pourrait faire reprendre la plus ancienne campagne. On voudra toujours récupérer la campagne à reprendre par contre on pourrait changer la règle de sélection de la campagne à reprendre.

 je proposerais plus un nom du genre 'participationToContinue', 'participationToFinish'